Symfony/CS/Tokenizer/Transformers.php

```diff
@@ -24,7 +24,7 @@ class Transformers
     /**
      * Array of registered Transformer classes.
      *
```

Comment:
not sure about the change, but there is no Transformer class. Maybe an array of objects implementing the TransformerInterface?